packages/edit-site/src/components/post-list/index.js

```diff
@@ -253,7 +253,7 @@ export default function PostList( { postType } ) {
 		// We want to provide a different default item for the status filter
 		// than the REST API provides.
 		if ( ! filters.status || filters.status === '' ) {
-			filters.status = DEFAULT_STATUSES;
+			// filters.status = DEFAULT_STATUSES;
```

Comment:
Just commenting this out for now as the media endpoint isn't happy being handed all the statuses that posts use (since it doesn't support all of them)